sklearn/linear_model/tests/test_ridge.py

```diff
@@ -726,6 +734,20 @@ def _test_ridge_classifiers(filter_):
     assert np.mean(y_iris == y_pred) >= 0.8
 
 
+@pytest.mark.parametrize("scoring", [None, "accuracy", _accuracy_callable])
+@pytest.mark.parametrize("cv", [None, KFold(5)])
+@pytest.mark.parametrize("filter_", [DENSE_FILTER, SPARSE_FILTER])
+def test_ridgeClassifier_with_scoring(filter_, scoring, cv):
+    # regression test for #14672
+    # check that RidgeClassifierCV works with all sort of scoring and
+    # cross-validation
+    scoring_ = make_scorer(scoring) if callable(scoring) else scoring
+    clf = RidgeClassifierCV(scoring=scoring_, cv=cv)
+    clf.fit(filter_(X_iris), y_iris)
+    y_pred = clf.predict(filter_(X_iris))
+    assert np.mean(y_iris == y_pred) >= 0.8
```

Comment:
this test is too weak. It should test the cv_scores and you can have a good prediction even if you screwed up the score and the problem is easy.

Reply:
This is only a non-regression test. Before this would fail. We can remove the assert and put a comment that this is a smoke test and not give any certainty regarding the score obtained?